data/production/animal-colors.json

```diff
@@ -0,0 +1,338 @@
+        "name": "Chocolate Brown",
+        "id": 1010910
+    },
+    {
+        "name": "Cinnamon",
+        "id": 1010911
+    },
+    {
+        "name": "Copper",
+        "id": 1010912
+    },
+    {
+        "name": "Cream",
+        "id": 1010913
+    },
+    {
+        "name": "Dapple",
+        "id": 1010914
+    },
+    {
+        "name": "Dappled",
+        "id": 1010915
+    },
+    {
+        "name": "Dead Grass",
+        "id": 1010916
+    },
+    {
+        "name": "Dun",
+        "id": 1010917
+    },
+    {
+        "name": "Fawn",
+        "id": 1010918
+    },
+    {
+        "name": "Flame Point",
+        "id": 1010919
+    },
+    {
+        "name": "Ginger",
+        "id": 1010920
+    },
+    {
+        "name": "Gold",
+        "id": 1010921
+    },
+    {
+        "name": "Gray",
+        "id": 1010922
+    },
+    {
+        "name": "Green",
+        "id": 1010923
+    },
+    {
+        "name": "Grey",
+        "id": 1010924
+    },
+    {
+        "name": "Grey Tabby",
+        "id": 1010925
+    },
+    {
+        "name": "Grey White",
+        "id": 1010926
+    },
+    {
+        "name": "Gruel",
+        "id": 1010927
+    },
+    {
+        "name": "Harlequin",
+        "id": 1010928
+    },
+    {
+        "name": "Honey",
+        "id": 1010929
+    },
+    {
+        "name": "Ivory",
+        "id": 1010930
+    },
+    {
+        "name": "Lemon",
+        "id": 1010931
+    },
+    {
+        "name": "Leopard Appaloosa",
+        "id": 1010932
+    },
+    {
+        "name": "Lilac Point",
+        "id": 1010933
+    },
+    {
+        "name": "Liver",
+        "id": 1010934
+    },
+    {
+        "name": "Lynx Point",
+        "id": 1010935
+    },
+    {
+        "name": "Mahogany",
+        "id": 1010936
+    },
+    {
+        "name": "Multi",
+        "id": 1010937
+    },
+    {
+        "name": "Orange",
+        "id": 1010938
+    },
+    {
+        "name": "Orange Tabby",
+        "id": 1010939
+    },
+    {
+        "name": "Other",
+        "id": 1010940
+    },
+    {
+        "name": "Paint",
+        "id": 1010941
+    },
+    {
+        "name": "Palamino",
+        "id": 1010942
+    },
+    {
+        "name": "Peach",
+        "id": 1010943
+    },
+    {
+        "name": "Piebald",
+        "id": 1010944
+    },
+    {
+        "name": "Pink",
+        "id": 1010945
+    },
+    {
+        "name": "Purple",
+        "id": 1010946
+    },
+    {
+        "name": "Red",
+        "id": 1010947
+    },
+    {
+        "name": "Reddish Brown",
+        "id": 1010948
+    },
+    {
+        "name": "Roan",
+        "id": 1010949
+    },
+    {
+        "name": "Rust",
+        "id": 1010950
+    },
+    {
+        "name": "Sable",
+        "id": 1010951
+    },
+    {
+        "name": "Sage",
+        "id": 1010952
+    },
+    {
+        "name": "Salt And Pepper",
+        "id": 1010953
+    },
+    {
+        "name": "Sand",
+        "id": 1010954
+    },
+    {
+        "name": "Seal Point  ",
```

Comment:
Spacing?  Is this important?

Reply:
I have no clue how you saw that, probably, but not noticeable. I will update this is another branch.